flink-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/internals/KafkaTopicsDescriptor.java

```diff
@@ -41,17 +42,25 @@
     private final Pattern topicPattern;
 
     public KafkaTopicsDescriptor(
-            @Nullable List<String> fixedTopics, @Nullable Pattern topicPattern) {
+        @Nullable List<String> fixedTopics, @Nullable Pattern topicPattern) {
```

Comment:
The code format is not right, have you follow the guide 
https://ci.apache.org/projects/flink/flink-docs-master/docs/flinkdev/ide_setup/#code-formatting ?
We need a consistent code format.